components/phpunit_bridge.rst

```diff
@@ -119,7 +119,27 @@ an arbitrary value (ex: ``320``) will make the tests fails only if a higher
 number of deprecation notices is reached (``0`` is the default value). You can
 also set the value ``"weak"`` which will make the bridge ignore any deprecation
 notices. This is useful to projects that must use deprecated interfaces for
-backward compatibility reasons.
+backward compatibility reasons, or for libraries that do not want deprecations
+triggered by their dependencies to prevent unrelated pull requests from being
+merged, which leads us to the ``"weak_vendors"`` value, described below.
```

Comment:
After more thinking, maybe you're right, I'm kind of explaining the same thing twice here.